addons/panku_console/common/config.gd

```diff
@@ -8,8 +8,12 @@ const OPTIONS = {
 	CUSTOM_DEFAULT_CONFIG = 'custom_default_config',
 }
 
-const INITIAL_DEFAULT_CONFIG_FILE_PATH = "res://addons/panku_console/default_panku_config.cfg"
-const USER_CONFIG_FILE_PATH = "user://panku_config.cfg"
+const INITIAL_DEFAULT_CONFIG_FILE_PATH = "res://addons/panku_console/default_panku_config.tres"
```

Comment:
I believe we need to change file extension hint [here](https://github.com/Ark2000/PankuConsole/blob/2346e730221b44a055e593721b12dae7306aeefd/addons/panku_console/plugin.gd#L84) too.

Also, do we need to keep `default_panku_config.cfg` file in project or may be we can delete it within this pull request?